lib/callmonitor.js

```diff
@@ -211,7 +211,7 @@ function CallMonitor(adapter, devices, phonebook) {
         });
 
         client.connect({
-            host: adapter.config.ip,
+            host: adapter.config.ip || adapter.config.iporhost,
```

Comment:
you need to switch the order here ... else "ip" (which is no longer editable but might stay set would win). So

adapter.config.iporhost || adapter.config.ip

would be right

Reply:
It is not needed to change the order, because the old setting will be resetted -> see index.html:309
change is comming soon

Reply:
I also saw that with the null reset ... ;-)